app/assets/stylesheets/v3/layout_components/responsive-table.scss

```diff
@@ -198,3 +200,80 @@ $use-respond-mixins: false !default;
     margin-bottom: 0;
   }
 }
+
+// This is the new table css as of Nov 2020. Please use this going forward.
+
+.responsive-table {
+  border-style: none;
+  border-collapse: collapse;
+  width: 100%;
+  table-layout: fixed;
+
+  thead,
+  tbody {
+    border: none;
+  }
+
+  th,
+  td {
+    padding: $base-padding;
+    text-align: left;
+    border-right: 1px solid $bg-grey-light;
```

Comment:
is there a reason we don't want to use the $base-border var for this?

Reply:
Because I'm dumb. 😛